src/scripts/scaling/scalingAction.sh

```diff
@@ -183,7 +194,8 @@ in_cluster_startup=`echo ${DOMAIN} | python cmds-$$.py`
 function get_num_ms_in_cluster() {
   local DOMAIN="$1"
   local num_ms
-  if [ -x "$(command -v jq)" ]; then
+  trace "zzz- wls_cluster_name is ${wls_cluster_name}"
```

Comment:
Does this trace need to be cleaned up?

Reply:
Yes, it will be cleaned up. This is WIP and I am still adding more tests and making changes to scalingAction.sh as needed.